tests/vars-iri.yaml

```diff
@@ -1,7 +0,0 @@
-# Variable overrides for testing at IRI.
-
-# Public NTP servers are blocked inside the IRI network, so
-# without this override the chronyc test fails when run at IRI.
```

Comment:
Why did you delete this file? If you're having trouble with the crony tests, I think this is why. If you don't override the ntp server list, crony on the vagrant guest tries to contact outside NTP servers, which are blocked (I assume by the Lamont firewall).